cli/pcluster/config/param_types.py

```diff
@@ -19,6 +19,7 @@
 
 from pcluster.config.update_policy import UpdatePolicy
 from pcluster.config.validators import settings_validator
+from pcluster.utils import get_file_section_name
```

Comment:
Why are we moving this function?

I don't like too much our utils module, it is full of totally different functions, with different scopes.
I'm more in favour of trying to keep the utility functions in the related modules.

What do you think?

Reply:
I had to move it because I needed to use it from the validator, and this caused a circular reference problem. I agree in the future we should organize better the utils function.